source/auth/auth.rst

```diff
@@ -542,7 +557,7 @@ Auth Related Options
 authMechanism
 	MONGODB-CR, MONGODB-X509, GSSAPI, PLAIN, SCRAM-SHA-1, SCRAM-SHA-256
 
-	Sets the Mechanism property on the MongoCredential. The default is determined based on server version and user credentials.
+	Sets the Mechanism property on the MongoCredential. The default is determined based on server version, user credentials, and the ``saslSupportedMechs`` field of the server's isMaster reply.
```

Comment:
... now the default is determined *not* based on server version.

Reply:
It's a mix of all three.  Maybe I'll strip details and just say default is negotiated.